rdflib/plugins/stores/sparqlstore.py

```diff
@@ -145,6 +109,84 @@ def CastToTerm(node):
         raise Exception('Unknown answer type')
 
 
+def CastToTerm(node):
+    warnings.warn(
+        "Call to deprecated function CastToTerm, use _node_from_result.",
+        category=DeprecationWarning,
+    )
+    return _node_from_result(node)
+
+
+def _node_to_sparql(node):
+    if isinstance(node, BNode):
+        raise Exception(
+            "SPARQLStore does not support BNodes! "
```

Comment:
Maybe this Exception should now say "SPARQLStore does not support BNodes *by default*! See ..."